test/amqp-integration-test.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, cmd, proc):
 def run(path):
     global processes
     binary = os.path.join(tempdir, os.path.basename(path))
-    cmd = 'go build -tags pkcs11 -o %s %s' % (binary, path)
+    cmd = 'GORACE="halt_on_error=1" go build -tags pkcs11 -race -o %s %s' % (binary, path)
```

Comment:
(Willing to do it if y'all want it, but just wanted to make sure y'all were aware of why I did it this way.)